titan-compiler/checker.lua

```diff
@@ -740,18 +732,21 @@ end
 --   ast: AST for the whole module
 --   subject: the string that generated the AST
 --   filename: the file name that contains the subject
---   returns true if typechecking succeeds, or false and a list of type errors found
+--   loader: the module loader, a function from module name to its AST, code,
+--   and filename or nil and an error
+--
+--   returns true if typechecking succeeds, or false and a list of type errors
+--   found
 --   annotates the AST with the types of its terms in "_type" fields
 --   annotates duplicate top-level declarations with a "_ignore" boolean field
---   loader: the module loader, a function from module name to its AST, code, and filename or nil and an error
 function checker.check(modname, ast, subject, filename, loader)
-    loader = loader or function () return nil, "you must pass a loder to import modules" end
+    loader = loader or function ()
+        return nil, "you must pass a loder to import modules"
```

Comment:
Did we already tell Hisham that we currently use a *lode* to *run* our modules? :)

Perhaps we should take this opportunity to fix this old typo here.

Reply:
hahaha, or rename the variable to `loderunner` 😆

Reply:
Done :)